src/client/Microsoft.Identity.Client/TokenResponseHelper.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Instance;
+using Microsoft.Identity.Client.Internal;
+using Microsoft.Identity.Client.Internal.Requests;
+using Microsoft.Identity.Client.OAuth2;
+
+namespace Microsoft.Identity.Client
+{
+    internal class TokenResponseHelper
```

Comment:
is this all refactoring or did you add any new logic?

Reply:
all refactoring